src/server/api/routers/user.ts

```diff
@@ -92,7 +92,16 @@ export const userRouter = createTRPCRouter({
 
       return triedRemovingLastAdmin;
     }),
-
+  selectedUser: publicProcedure
+    .input(
+      z
+        .object({
+          text: z.string(),
```

Comment:
Can you please rename this to `userId`, `text` is really deceptive

Reply:
done